ultet/src/test/scala/za/co/absa/ultet/parsers/PgFunctionFileParserTest.scala

```diff
@@ -28,12 +30,12 @@ class PgFunctionFileParserTest extends AnyFlatSpec with Matchers {
       |--      [Descrip""".stripMargin
 
     PgFunctionFileParser().parseString(functionString) shouldBe DBFunctionFromSource(
-      fnName = "public_functionX",
-      paramTypes = Seq("TEXT", "INTEGER", "hstore"),
-      owner = "owner_user123",
-      users = Seq("user_for_accessA", "user_for_accessB", "user_for_accessZ"),
-      schema = "my_schema1",
-      database = "eXample_db",
+      fnName = FunctionName("public_functionX"),
```

Comment:
These changes will probably conflict, because they have been done in #92.

Reply:
okay thanks, gonna resolve now